app/models/la_funded_place.rb

```diff
@@ -1,5 +1,9 @@
 class LaFundedPlace < CompulsorySchool
   def institution_type
-    'funded_places'
+    'local authority'
+  end
+
+  def techsource_urn
+    "iss #{responsible_body.name.downcase}"
```

Comment:
```suggestion
    "iss#{responsible_body.name.downcase}"
```

Probably safer to not have a space here as it could impact CC downstream.

Reply:
Hmm yes, although the RB names might have spaces too, maybe we need something more robust. Also I'm not sure this has been agreed with CC yet?

Reply:
No, I think we would need to check what they'd like in this instance @markhallen

Reply:
Also this method is only used in a view to playback to a user. Does it not need to hook in somewhere where we need to provide this info to CC so they can correlate things?

Reply:
Yes it will have to at some point

Reply:
Just fyi - @ralph-hawkins had highlighted that we need to differentiate between ISS and Social Care and had suggested the __iss-__ and __slc-__ prefix